src/System.Net.Security/tests/FunctionalTests/DummyTcpServer.cs

```diff
@@ -140,40 +143,25 @@ private void OnAccept(IAsyncResult result)
                 if (useSsl)
                 {
                     state = new ClientState(client, sslEncryptionPolicy);
-                    Logger.LogInformation("Server: starting SSL authentication.");
+                    _log.WriteLine("Server: starting SSL authentication.");
 
 
                     SslStream sslStream = null;
                     X509Certificate2 certificate;
+                    certificate = new X509Certificate2("DummyTcpServer.pfx");
```

Comment:
PEM is not supported by .Net and so I can't use the Base64 format for private-key certs.
I am updating the PR by referencing the same package as the one used by X509CertificatesTests. Since we already had this cert with null password, it's now in use by those tests.